lib/api/kuzzle.js

```diff
@@ -204,7 +223,7 @@ function registerErrorHandlers(kuzzle) {
   });
 
   process.removeAllListeners('uncaughtException');
-  process.on('uncaughtException', (err) => {
+  process.on('uncaughtException', err => {
     console.error(`ERROR: uncaughtException: ${err.message}`, err.stack); // eslint-disable-line no-console
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 107 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComment)